entities/Chef.java

```diff
@@ -0,0 +1,67 @@
+package entities;
+
+import java.util.List;
+
+public class Chef extends Base {
+
+    //Attributes
+
+    private String firstName;
+    private String lastName;
+    private Integer experience;
+    private List<Order> assignedOrder;
+
+    //Constructors
+
+    public Chef() {
+    }
+
+    public Chef(String firstName, String lastName, Integer experience, List<Order> assignedOrder) {
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.experience = experience;
+        this.assignedOrder = assignedOrder;
+    }
+
+    //Methods
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    public Integer getExperience() {
+        return experience;
+    }
+
+    public void setExperience(Integer experience) {
+        this.experience = experience;
+    }
+
+    public List<Order> getAssignedOrder() {
+        return assignedOrder;
+    }
+
+    public void setAssignedOrder(List<Order> assignedOrder) {
+        this.assignedOrder = assignedOrder;
+    }
+
+    public void addAssignedOrder(Order order) {
+        assignedOrder.add(order);
+    }
```

Comment:
Don't this in an entity. An entity can not do this kind of operations, it should only hold / store information

Reply:
Done